scripts/test

```diff
@@ -20,10 +33,12 @@ if [[ ${ARCH} == amd64 ]]; then
     go test -cover -tags=test ./...
 
     k3s server >/dev/null 2>&1 &
-    sleep 10
-
     export KUBECONFIG=/etc/rancher/k3s/k3s.yaml
-    export PATH=$(pwd)/bin:$PATH
+    waitUntilk3sIsReady
+
+    k3s kubectl apply -f ./scripts/rbac.yaml
+    PATH=$(pwd)/bin:$PATH
```

Comment:
Any reason for the split ?

Reply:
The bash linter recommended separating the assignment from the export to prevent value masking.